light/lightchain.go

```diff
@@ -389,6 +389,21 @@ func (self *LightChain) CurrentHeader() *types.Header {
 	return self.hc.CurrentHeader()
 }
 
+// CurrentBlock exists for interface compatibility and always returns nil
+func (self *LightChain) CurrentBlock() *types.Block {
+	return nil
+}
+
+// CurrentFastBlock exists for interface compatibility and always returns nil
+func (self *LightChain) CurrentFastBlock() *types.Block {
+	return nil
+}
+
+// FastSyncCommitHead exists for interface compatibility and does nothing
+func (self *LightChain) FastSyncCommitHead(h common.Hash) error {
+	return nil
+}
```

Comment:
I'd really like to avoid adding these, perhaps by doing a similar magic that I suggested for the `downloader.LightPeerConnection` and `downloader.FullPeerConnection` above?